internal/pkg/earpointer/analysis_test.go

```diff
@@ -1068,15 +1080,23 @@ func TestVariadicCall(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	want := concat(map[string]string{
-		"{f.a,f.t4,g.t1}":  "[]",
-		"{f.b}":            "[]",
-		"{f.t0,f.t3,g.ks}": "[0->g.t0, 1->f.t2]",
-		"{f.t1,g.t0}":      "--> g.t1",
-		"{f.t2}":           "--> f.b",
+	// Handle the non-determinism when choosing the representative during unifying.
```

Comment:
Consider highlighting the part of the `want` that can change due to non-determinism with e.g. a comment.